docs/ADD_NEW_EXCHANGE.md

```diff
@@ -0,0 +1,1064 @@
+git clone https://github.com/thrasher-corp/gocryptotrader.git
+cd gocryptotrader/cmd/documentation/exchange_template
+go run exchange_template.go -name FTX -ws -rest
+```
+
+#### Windows
+
+```bash
+git clone https://github.com/thrasher-corp/gocryptotrader.git
+cd gocryptotrader\cmd\exchange_template
+go run exchange_template.go -name FTX -ws -rest
+```
+
+### Add exchange struct to (config_example.json)[../config_example.json], (configtest.json)[../testdata/configtest.json]:
+
+Find out which asset types are supported by the exchange and add them to the pairs struct (spot is enabled by default)
+
+#### If main config path is unknown the following function can be used:
+```go
+config.GetDefaultFilePath()
+```
+
+```go
+  {
+   "name": "FTX",
+   "enabled": true,
+   "verbose": false,
+   "httpTimeout": 15000000000,
+   "websocketResponseCheckTimeout": 30000000,
+   "websocketResponseMaxLimit": 7000000000,
+   "websocketTrafficTimeout": 30000000000,
+   "websocketOrderbookBufferLimit": 5,
+   "baseCurrencies": "USD",
+   "currencyPairs": {
+    "assetTypes": [
+      "spot",
+      "futures"
+     ],
+     "pairs": {
+      "futures": {
+       "enabled": "BTC-PERP",
+       "available": "BTC-PERP",
+       "requestFormat": {
+        "uppercase": true,
+        "delimiter": "-"
+       },
+       "configFormat": {
+        "uppercase": true,
+        "delimiter": "-"
+       }
+      },
+      "spot": {
+       "enabled": "BTC/USD",
+       "available": "BTC/USD",
+       "requestFormat": {
+        "uppercase": true,
+        "delimiter": "/"
+       },
+       "configFormat": {
+        "uppercase": true,
+        "delimiter": "/"
+       }
+      }
+     }
+    },
+   "api": {
+    "authenticatedSupport": false,
+    "authenticatedWebsocketApiSupport": false,
+    "endpoints": {
+     "url": "NON_DEFAULT_HTTP_LINK_TO_EXCHANGE_API",
+     "urlSecondary": "NON_DEFAULT_HTTP_LINK_TO_EXCHANGE_API",
+     "websocketURL": "NON_DEFAULT_HTTP_LINK_TO_WEBSOCKET_EXCHANGE_API"
+    },
+    "credentials": {
+     "key": "Key",
+     "secret": "Secret"
+    },
+    "credentialsValidator": {
+     "requiresKey": true,
+     "requiresSecret": true
+    }
+   },
+   "features": {
+    "supports": {
+     "restAPI": true,
+     "restCapabilities": {
+      "tickerBatching": true,
+      "autoPairUpdates": true
+     },
+     "websocketAPI": true,
+     "websocketCapabilities": {}
+    },
+    "enabled": {
+     "autoPairUpdates": true,
+     "websocketAPI": false
+    }
+   },
+   "bankAccounts": [
+    {
+     "enabled": false,
+     "bankName": "",
+     "bankAddress": "",
+     "bankPostalCode": "",
+     "bankPostalCity": "",
+     "bankCountry": "",
+     "accountName": "",
+     "accountNumber": "",
+     "swiftCode": "",
+     "iban": "",
+     "supportedCurrencies": ""
+    }
+   ]
+  },
+```
+
+#### Available pairs will be automatically filled in the configs when wrapper functions are filled out and gocryptotrader is run with the new exchange enabled
```

Comment:
Change this to advise that the user can run `./gocryptotrader --config=config_example.json` to automatically fill these out, but have to make sure that they don't override the ntp client and encrypt config default settings.